root/etc/s6-overlay/s6-rc.d/init-pyload-config/run

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/with-contenv bash
+# shellcheck shell=bash
 
 # create our folders
 mkdir -p \
     /config/settings \
     /downloads
 
 # default config file
-[[ ! -f "/config/settings/pyload.cfg" ]] && \
-    cp \
-        /defaults/pyload.cfg \
-        /config/settings/pyload.cfg
+cp -n \
+    /defaults/pyload.cfg \
+    /config/settings/pyload.cfg
 
 # permissions
 echo "[cont-init.d] Setting permissions this may take some time"
```

Comment:
This bit probably isn't necessary, we don't do it anywhere else and it's not cont-init.d any more.

Reply:
Removed that line (in both PRs)